set.mm

```diff
@@ -527453,6 +527584,45 @@ C Fn ( ( S i^i dom F ) u. { z } ) ) $=
       JYRUXKUWEUYMUXKUWBUUPUWCUUSUWDUVTCAWRUVTCBWRYJYKUJYSUUAUUBUUCUUDVJUURUVAU
       VBUUTSZUUTUVAUVBSZUURUUHVUDUWLBCUUEUHVUDUVBUUTUVASVUEUVAUVBUUTYTUVBUUTUVA
       YTRWDUUF $.
+
+    $( Given ` A ` less than ` B ` , equal to ` B ` up to ` X ` , and undefined
+       at ` X ` , then ` A ( X ) = 1o ` .  (Contributed by Scott Fenton,
+       9-Aug-2024.) $)
+    nogt01o $p |- ( ( ( A e. No /\ B e. No /\ X e. On ) /\
+    ( ( A |` X ) = ( B |` X ) /\ A <s B ) /\ ( B ` X ) = (/) ) ->
+    ( A ` X ) = 1o ) $=
```

Comment:
` B ` is undefined at ` X ` rather than ` A `, so maybe reword (it also matches the label)...
```suggestion
    $( Given ` B ` greater than ` A ` , equal to ` A ` up to ` X ` , and
       undefined at ` X ` , then ` A ( X ) = 1o ` .  (Contributed by Scott
       Fenton, 9-Aug-2024.) $)
    nogt01o $p |- ( ( ( A e. No /\ B e. No /\ X e. On ) /\
    ( ( A |` X ) = ( B |` X ) /\ A <s B ) /\ ( B ` X ) = (/) ) ->
    ( A ` X ) = 1o ) $=
```